### PR TITLE
Trial Run: Add people

### PR DESCRIPTION
**Pull request by NPDebs**
This PR shows an attempt to add people to the website, guided by the beautifully-documented [Knowledge Management System](https://openlifesci.org/knowledge_management.html) (thank you, Bérénice!).

B.B provided a `.csv file` from civi containing some new community members.
With this,
- I set up a virtual environment, 
- installed dependencies, 
- ran the python scripts and 
- added new people to the website.

_Success Screenshots!_ 🎉
add_people

add_people2

add_people3

**Note for reviewers:** The plugins are making it take a bit longer to serve up the site...and it is trying to kill my computer. 😆😭

### CODE REVIEW

File: _data/people.yaml [hunk 1]
```diff
@@ -6453,6 +6459,12 @@ mgawe-cavin:
   continent: Africa
   longitude: 36.8288420082562
   latitude: -1.3026148499999999
+mgomezn:
+  first-name: MARIANA PATRICIA
+  last-name: GOMEZ NICOLAS
```
Comment (bebatut) [inline _data/people.yaml:6464]: Can you put the names as capitalized (so only 1st letter as upper case)?
Note for latter: we need to modify the script so it does it

**Review by bebatut — COMMENTED** [review]
Nice @NPDebs!

File: _data/people.yaml (hunk 1 above)
Comment (NPDebs) [reply _data/people.yaml:6464]: For sure! 💯

Afterwards, I can also modify the scripts and PR on a different branch.

File: _data/people.yaml [hunk 2]
```diff
@@ -6453,6 +6459,12 @@ mgawe-cavin:
   continent: Africa
   longitude: 36.8288420082562
   latitude: -1.3026148499999999
+mgomezn:
+  first-name: MARIANA PATRICIA
```
Comment (NPDebs) [inline _data/people.yaml:6463]:
```suggestion
  first-name: Mariana Patricia
```

File: _data/people.yaml (hunk 1 above)
Comment (NPDebs) [inline _data/people.yaml:6464]:
```suggestion
  last-name: Gomez Nicolas
```

**Comment by bebatut** [comment]
Thanks @NPDebs

**Pull request merged**